README.md

```diff
@@ -1 +1,118 @@
-# AwesomeBookES6
\ No newline at end of file
+# AwesomeBookES6
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+**[Awesome Books]** is a website that stores the books with their title and authors respectively. A user is able to add a new book and delete that book 
+
+## 🛠 Built With <a name="built-with"></a>
+
+-HTML, CSS, JavaScript, Linters, and GitHub
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://css.com/">CSS</a></li>
+    <li><a href="https://javascript.com/">JavaScript</a></li>
+    <li>Technologies used Git, Github, and linters</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[The homepage]**
+- [ ] **[JavaScript Functionality]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://jessekay.github.io/AwesomeBooksES6/)
```

Comment:
- [ ] Live demo link is broken. Kindly fix the link so that the live demo can be accessed to test the project's functionality. **NOTE**: The live demo can be deleted if its not a project requirement to deploy the project 🤝